sdk/trace/span.go

```diff
@@ -298,6 +316,26 @@ func typeStr(i interface{}) string {
 	return fmt.Sprintf("%s.%s", t.PkgPath(), t.Name())
 }
 
+func recordStackTrace(err error) string {
+	var stackTrace []uintptr
+	var convertedStackTrace string
+
+	stackTrace = make([]uintptr, 32)
+	n := runtime.Callers(1, stackTrace)
+	stackTrace = stackTrace[:n]
+
+	frames := runtime.CallersFrames(stackTrace)
+	d := true
+	for frame, more := frames.Next(); d; frame, more = frames.Next() {
+		if !strings.Contains(frame.Function, "runtime.") && !strings.Contains(frame.Function, "testing.") {
```

Comment:
Why would this omit data about `runtime` or `testing` functions?  More generally, do we need to construct a stack trace manually like this or can we just use the result from calling [`runtime.Stack()`](https://pkg.go.dev/runtime#Stack)?

Reply:
fair. Added use of `runtime.Stack()` to capture stack traces.